cms/static/js/views/video/transcripts/file_uploader.js

```diff
@@ -31,7 +31,7 @@ function($, Backbone, _, Utils) {
 
             if (tplContainer.length) {
                 if (!tpl) {
-                    console.error('Couldn\'t load Transcripts File Upload template');
+                    console.error(gettext("Couldn't load Transcripts File Upload template"));
```

Comment:
This is another thing I'm not convinced should be i18n'd. This will appear in the JS console when errors occur; it will be impossible for a developer to find the place this happened in the code if it's translated, especially fi the error report comes from a user.